### PR TITLE
Initial bikeshed file

### PR DESCRIPTION
**Pull request by mukul-p**

### CODE REVIEW

**Comment by mukul-p** [comment]
For those who are new - This would be (snipped) output of the current config - 
image

**Review by Jack-Works — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft
```
Comment (hober) [inline index.bs]: s/ Draft//

File: index.bs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft
+Shortname: webextensions
+Level: 1
+Status: w3c/ED
```
Comment (hober) [inline index.bs]: s|w3c/ED|CG-DRAFT|

File: index.bs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft
+Shortname: webextensions
+Level: 1
+Status: w3c/ED
+Group: w3c/webextensions
```
Comment (hober) [inline index.bs]: s|w3c/webextensions|WECG|

File: index.bs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft
+Shortname: webextensions
+Level: 1
+Status: w3c/ED
+Group: w3c/webextensions
+URL: https://github.com/w3c/webextensions
```
Comment (hober) [inline index.bs]: the URL should be https://w3c.github.io/webextensions/

File: index.bs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft
+Shortname: webextensions
+Level: 1
+Status: w3c/ED
+Group: w3c/webextensions
+URL: https://github.com/w3c/webextensions
+Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
+Editor: Tomislav Jovanovic, Mozilla, https://www.mozilla.org/, tjovanovic@mozilla.com
```
Comment (hober) [inline index.bs]: These two Editor lines use different syntax; have you tried this in Bikeshed and do they produce the same result?

File: index.bs (hunk 3 above)
Comment (hober) [reply index.bs]: Note that this will require a corresponding change to Bikeshed, which I'll submit a PR for today.

File: index.bs (hunk 1 above)
Comment (hober) [reply index.bs]: (It's unnecessary to say that it's a draft in the title, as the Status metadata item establishes that.)

**Review by hober — CHANGES_REQUESTED** [review]

File: index.bs (hunk 3 above)
Comment (hober) [reply index.bs]: https://github.com/tabatkins/bikeshed/pull/2111

Comment (hober) [reply index.bs]: nota bene https://github.com/tabatkins/bikeshed/pull/2111#issuecomment-880984568

File: index.bs (hunk 5 above)
Comment (mukul-p) [reply index.bs]: Indeed there was a difference (because of one comma). Thanks

**Review by hober — APPROVED** [review]
Looks good to me!

**Pull request merged**